packages/compass-components/src/components/options-dropdown-button.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useMemo } from 'react';
+import { Link, Icon } from './leafygreen';
+import { css } from '@leafygreen-ui/emotion';
+import { spacing } from '@leafygreen-ui/tokens';
+import { uiColors } from '@leafygreen-ui/palette';
+
+const optionsButtonStyles = css({
+  backgroundColor: 'transparent',
+  border: 'none',
+  display: 'inline',
+  height: spacing[4] + spacing[1],
+  ':focus': {
+    outline: `${spacing[1]}px auto ${uiColors.focus}`,
+  },
```

Comment:
This doesn't look aligned with leafygreen: the one they have has border radius and a different spacing I think

Reply:
Going to `useFocusRing`